File: 1/1.js
function multiplesOf3and5(number) {

  let ans=0
  for (let i = 0; i < number; i++) {
    if(i%3==0 || i%5==0)
      ans+=i;
} 
  console.log(ans)
  return ans
}

multiplesOf3and5(10);

File: 1/1.py
import time
st = time.time()
lim = 1000
ans = 0
for i in range(lim):
    if i % 3 == 0 or i % 5 == 0:
        ans += i
print('ans =', ans)
print(time.time() - st, 's')
